.github/workflows/certify.yml

```diff
@@ -0,0 +1,28 @@
+name: Manually certify all container images
+
+on:
+  workflow_dispatch:
+ 
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  build:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v4
+      - name: Login to Open Source ECR
+        run: make login
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+      - name: Get preflight
+        run: |
+          curl -LO https://github.com/redhat-openshift-ecosystem/openshift-preflight/releases/latest/download/preflight-linux-amd64
+          chmod +x preflight-linux-amd64
+          mv preflight-linux-amd64 /usr/local/bin/preflight
```

Comment:
Can we add a make target to install this locally, maybe in `./bin/`, and then use that in the CI here? I'd like to make this as easy as possible to run locally.

Reply:
changed